scripts/build.py

```diff
@@ -110,6 +110,8 @@ def get_target_platforms():
 ANDROID_BUILD_TOOLS_VERSION="32.0.0"
 PACKAGES_CCTOOLS_PORT="cctools-port-darwin19-6c438753d2252274678d3e0839270045698c159b-linux"
 
+PIP_VERSION = "22.2.2"
```

Comment:
Why do we need to use a specific version with Python3?
We needed it before, with python 2, in order to make the current wheels work with the exe.

Now I would guess we shouldn't specify a version, but just go with the default version.

Reply:
Removed 👍